test/fixtures/failures.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+exports.inputs = [
+  () => 'https://google.com',
+  () => 'git+ssh://git@nothosted.com/abc/def',
+  () => 'git://nothosted.com',
+  () => 'git+file:///foo/bar',
+  () => 'completely-invalid',
+  () => 'git+ssh://git@git.unlucky.com:RND/electron-tools/some-tool#2.0.1',
+  () => 'git://bitbucket.org/foo/bar', // unsupported protocol
+  () => '::',
+  () => '',
+  () => null,
+  () => undefined
+]
+
+exports.params = [
+  {}
+]
+
+/* these need to be ported
```

Comment:
Did this get done?

Reply:
nope! good catch, forgot about those. I'll get them added in